assets/constants.ts

```diff
@@ -46,6 +46,12 @@ export = {
   // The default language code for an exploration.
   "DEFAULT_LANGUAGE_CODE": "en",
 
+  // The current version of the exploration states blob schema. If any backward-
+  // incompatible changes are made to the states blob schema in the data store,
+  // this version number must be changed and the exploration migration job
+  // executed.
+  "CURRENT_STATE_SCHEMA_VERSION": 30,
```

Comment:
I added this here since otherwise, validation will fail when creating the question suggestion [here](https://github.com/oppia/oppia/blob/develop/core/domain/suggestion_registry.py#L542).

Reply:
Note [this](https://github.com/oppia/oppia/pull/8057/files#r357962842) comment.